### ISSUE TITLE
null on Track.GetAlbumArt(AlbumArtSize size)

### ISSUE THREAD

**Issue by Pukka922** [JohnnyCrazy/SpotifyAPI-NET#80]
When i try to get the albumArt as Bitmap it always returns null.

track.GetAlbumArtUrl(AlbumArtSize size) returns a empty String

(Version 2.7.1)

**Comment by JohnnyCrazy** [comment]
Hi, did you use a nuget-package version or the current `master` branch? #79 fixed this, but a new nuget package isn't released yet because of some Web-API issues.

**Comment by Pukka922** [comment]
Thanks for this fast answer.

So i will wait for the next nuget update

**Comment by erinxocon** [comment]
@JohnnyCrazy Is this anything to do with this package, or is it on spotify's side?  I have time time to take a look if it's something that needs to be done to this project.

**Comment by JohnnyCrazy** [comment]
It's related to https://github.com/spotify/web-api/issues/199 (wrong error-responses of the web-api), so nothing we can do for now.

**Comment by Yukinii** [comment]
Just came around that issue myself, ended up getting it from the REST api instead.

`using (var cli = new WebClient())
            {
                var url = "https://api.spotify.com/v1/tracks/" + GetTrackURI().Split(':')[2];
                var json = cli.DownloadString(url);
                var data = JsonConvert.DeserializeObject<RootObject>(json); 
                //dynamic data = JsonConvert.DeserializeObject(json);
                
                return data.album.images[1].url; //0 = 640, 1 = 320, 2 = 64
            }`

**Comment by JohnnyCrazy** [comment]
@Yukinii 

You can also use the included Web-API Wrapper :wink: 

````
var spotify = new SpotifyWebAPI() { UseAuth = false };
var track = spotify.GetTrack("3Hvu1pq89D4R0lyPBoujSv");
return track.Album.Images[0].Url;
````

**Comment by JohnnyCrazy** [comment]
Just released version [2.8.0](https://github.com/JohnnyCrazy/SpotifyAPI-NET/releases/tag/2.8.0), so `GetAlbumArt` should be fixed now
